src/02-common-principles.md

```diff
@@ -542,14 +542,25 @@ Describing dates and timestamps:
 -   Date time information MUST be expressed in the following format
     `YYYY-MM-DDThh:mm:ss[.000000]` (year, month, day, hour (24h), minute,
     second, and optionally fractional second).
-    This is equivalent to the RFC3339 "date-time" format, time zone is always
-    assumed as local time).
+    This is almost equivalent to the [RFC3339](https://tools.ietf.org/html/rfc3339)
+    "date-time" format, with the exception that in BIDS, time zone is always
+    assumed as local time.
+    The RFC3339 `Z` syntax to indicate UTC or the RFC3339 "offset" syntax are not
+    supported.
     No specific precision is required for fractional seconds, but the precision
     SHOULD be consistent across the dataset.
-    For example `2009-06-15T13:45:30`
+    For example `2009-06-15T13:45:30`.
 
 -   Time stamp information MUST be expressed in the following format:
-    `13:45:30[.000000]`
+    `hh:mm:ss[.000000]`
+    For example `13:45:30`.
+
+-   Note that depending on local REB/IRB ethics board policy, Date time information
+    does not need to be fully detailed.
+    For example, it is permissible to set the time to `00:00:00` if reporting the
+    exact recording time is undesirable.
+    However, for privacy protection reasons, it it generally RECOMMENDED to shift
+    dates, as described below.
```

Comment:
```suggestion
    However, for privacy protection reasons, it is generally RECOMMENDED to shift
    dates, as described below, without completely removing time information, as time information can be useful for research purposes.
```

It doesn't have to be that, exactly, but the way that sentence was phrased was a little confusing. I just want to make it clear that the "however" refers to removing all time information vs. just anonymizing dates.

Reply:
Thanks @tsalo